packages/noir-ethereum-api/src/noir/oracles/accountOracles.ts

```diff
@@ -1,8 +1,9 @@
 import { type ForeignCallOutput } from '@noir-lang/noir_js';
 import { fromRlp, type GetProofReturnType, type Hex, isHex, type PublicClient } from 'viem';
 import { assert } from '../../assert.js';
-import { decodeHexAddress, encodeField, encodeHex } from '../encode.js';
+import { decodeField, decodeHexAddress, encodeField, encodeHex } from '../encode.js';
 import { padArray } from '../../arrays.js';
+import { all } from 'ramda';
```

Comment:
Any library with this function would be ok for me: lodash, underscore, ramda, etc. I propose ramda as it is focused on functional programming and offers immutability and side-effect free functions.

Reply:
Why would we need a library for that? You can write in pure js:
```js
array.every(isHex)
```

Reply:
right, replaced with native `every` method